src/main/java/com/oneandone/snmpman/configuration/AgentConfiguration.java

```diff
@@ -34,7 +34,7 @@
     public static class DeviceFactory {
 
         /** The default device which will be returned if no configuration specified. */
-        public static final Device DEFAULT_DEVICE = new Device("default", new Modifier[0]);
+        static final Device DEFAULT_DEVICE = new Device("default", new Modifier[0], null);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/1and1_NDev/snmpman/file/2405103230/issues/source?bid=3233453&fileBranchId=3233458#l37)